docs/ui/components/Tag/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { css } from '@emotion/react';
+import { borderRadius, spacing, theme } from '@expo/styleguide';
+
+import { formatName, getPlatformName, getTagStyle } from './helpers';
+
+import { B } from '~/components/base/paragraph';
+import { PlatformIcon } from '~/components/plugins/PlatformIcon';
+import { PlatformName } from '~/types/common';
+
+type TagProps = {
+  name: string;
+  firstElement?: boolean;
+  toc?: boolean;
+};
+
+type PlatformTagProps = Omit<TagProps, 'name'> & {
+  platform: PlatformName;
+};
+
+type StatusTagProps = Omit<TagProps, 'name'> & {
+  status: 'deprecated' | 'experimental' | string;
+};
+
+type PlatformTagsProps = {
+  prefix?: string;
+  platforms?: PlatformName[];
+  firstElement?: boolean;
+};
+
+export const Tag = ({ name, ...rest }: TagProps) => {
```

Comment:
Can we make it one component per file?
And also have the components defined like:
```jsx
function Tag(props: Props) {
  const { name, ...rest } = props;
   // ...
}
```

These are going to be small components, but that seems good to me. If we want them all exported from the same file, we can do:

```jsx
export { Tag } from './Tag';
export { Another } from './Another';
```

Reply:
Yeah, we us this pattern in most of new components, but since all of the tag components reuse same code, I thought it would be cleaner to keep everything in one file. Will refactor this then.